src/managedwriter/writer.ts

```diff
@@ -160,7 +160,7 @@ export class Writer {
     offsetValue?: IInt64Value['value']
   ): PendingWrite {
     let offset: AppendRowRequest['offset'];
-    if (offsetValue) {
+    if (offsetValue || offsetValue === 0) {
```

Comment:
Why is this needed?

Reply:
because when sending explicitly 0 as offset, this was evaluating as `false` and not being send